superset-frontend/src/components/Slider/index.tsx

```diff
@@ -16,13 +16,19 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import AntdSlider, {
-  SliderSingleProps,
-  SliderRangeProps,
-} from 'antd/lib/slider';
+import { SliderSingleProps, SliderRangeProps } from 'antd-v5/lib/slider';
+import { Slider as AntdSlider } from 'antd-v5';
+import { SupersetTheme, css } from '@superset-ui/core';
 
 export type { SliderSingleProps, SliderRangeProps };
 
 export default function Slider(props: SliderSingleProps | SliderRangeProps) {
-  return <AntdSlider {...props} css={{ marginLeft: 0, marginRight: 0 }} />;
+  return (
+    <AntdSlider
+      {...props}
+      css={(theme: SupersetTheme) => css`
+        margin: ${theme.gridUnit * 2 + 2}px 0 0 ${theme.gridUnit * 2 + 2}px;
```

Comment:
Can we only use full grid units?

Reply:
This was done to mimic the previous behavior of the component. If we don't force it like this there will be a non-negligible difference in margins.

Reply:
Yeah I know and... I guess it's fine? Unless those differences actually break something somewhere

Reply:
I think, if we are good with visual differences we can omit the custom style entirely and rely on what Antdesign provides out of the box.